### PR TITLE
New color themes, improved screen's UI

### PR DESCRIPTION
**Pull request by aquilesTrindade**
**New feature**: Change App Theme Options in ModSettings 
   ThemedActivity Activity created to serve as a basis for Activities that will support the theme 
**Redesign**: Redesigned Mod Settings activity
   Switch changed to MaterialSwitch 
   Toolbar changed to MaterialToolbar + CollapsingToolbar 
**Redesign**: Toolbar of Tools activity changed to CollapsingToolbar 
**Redesign**: SystemSettingsActivity
     used switch_layout(new file) for MaterialSwitch  
     used toolbar_m3(new file) for toolbar
**Redesign**: SystemInfoActivity  used New Toolbar M3

**About the deleted files:**
the deleted files: **themes.xml**, **styles.xml** 
These files were deleted because they were "replaced" with themes_blue.xml and styles_blue to be more organized. 

**Screenshots:**
Screenshot_20240504-012342_Sketchware Pro
Screenshot_20240504-012347_Sketchware Pro
Screenshot_20240504-012352_Sketchware Pro
Screenshot_20240504-012402_Sketchware Pro

### CODE REVIEW

File: app/src/main/java/com/besome/sketch/help/SystemSettingActivity.java [hunk 1]
```diff
@@ -23,12 +29,23 @@ public class SystemSettingActivity extends BaseAppCompatActivity {
...
-        switchItem.setName(Helper.getResString(resName));
-        switchItem.setDesc(Helper.getResString(resDescription));
-        switchItem.setValue(value);
-        contentLayout.addView(switchItem);
+        View switchLayout = LayoutInflater.from(this).inflate(R.layout.switch_layout, null);
+        MaterialSwitch materialSwitch = switchLayout.findViewById(R.id.material_switch);
+        materialSwitch.setChecked(value);
+        materialSwitch.setOnCheckedChangeListener((buttonView, isChecked) -> {
+            if (key == 0) {
+                preferenceEditor.putBoolean("P12I0", isChecked);
```
[the remark this answers is not in the document]
Comment (aquilesTrindade) [reply app/src/main/java/com/besome/sketch/help/SystemSettingActivity.java:37]: I don't understand, what do you want me to do?

File: app/src/main/java/mod/hilal/saif/activities/tools/ConfigActivity.java [hunk 2]
```diff
@@ -17,27 +15,59 @@
...
+import static mod.SketchwareUtil.dpToPx;
+import static mod.SketchwareUtil.getDip;
+import mod.SketchwareUtil;
+import mod.trindade.dev.theme.ThemedActivity;
+import mod.trindade.dev.theme.AppTheme;   
+ 
+public class ConfigActivity extends ThemedActivity {
 
-public class ConfigActivity extends Activity {
+    /* Material Design 3 by Aquiles Trindade on *29/04/2026* */
```
[the remark this answers is not in the document]
Comment (aquilesTrindade) [reply app/src/main/java/mod/hilal/saif/activities/tools/ConfigActivity.java]: removed!

File: app/src/main/java/mod/trindade/dev/theme/ThemedActivity.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package mod.trindade.dev.theme;
+
+import androidx.appcompat.app.AppCompatActivity; 
+import androidx.activity.EdgeToEdge;
+
+import android.os.Bundle;
+
+import mod.trindade.dev.theme.AppTheme;
+
+public class ThemedActivity extends AppCompatActivity {
```
[the remark this answers is not in the document]
Comment (aquilesTrindade) [reply app/src/main/java/mod/trindade/dev/theme/ThemedActivity.java:10]: why?

File: app/src/main/java/com/besome/sketch/help/SystemSettingActivity.java [hunk 4]
```diff
@@ -23,12 +29,23 @@ public class SystemSettingActivity extends BaseAppCompatActivity {
...
+            } else if (key == 1) {
+                preferenceEditor.putBoolean("P12I2", isChecked);
```
[the remark this answers is not in the document]
Comment (aquilesTrindade) [reply app/src/main/java/com/besome/sketch/help/SystemSettingActivity.java:39]: ...

File: app/src/main/java/mod/trindade/dev/theme/ThemedActivity.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (remaker17) [reply app/src/main/java/mod/trindade/dev/theme/ThemedActivity.java:10]: You literally don't know how to use interfaces?

[the remark this answers is not in the document]
Comment (aquilesTrindade) [reply app/src/main/java/mod/trindade/dev/theme/ThemedActivity.java:10]: I know, but not that much bro, are you going to judge me?

**Comment by PranavPurwar** [comment]
add proper padding to items in mod manager and change color of title vs description. Description should have light color.

**Comment by ilyassesalama** [comment]
May I know why do you keep closing your pull requests?

**Comment by aquilesTrindade** [comment]
> May I know why do you keep closing your pull requests?

 don't know.

**Comment by devsuay** [comment]
Very good, a shame they didn't accept it

**Comment by remaker17** [comment]
> Very good, a shame they didn't accept it 

We didn’t reject it, he closed the pull request before we checked it.

**Comment by devsuay** [comment]
> > Muito bom, pena que não aceitaram
> 
> Não rejeitamos, ele fechou o pull request antes de verificarmos.

oh, but would you accept it?

**Comment by PranavPurwar** [comment]
> > > Muito bom, pena que não aceitaram
> > 
> > 
> > Não rejeitamos, ele fechou o pull request antes de verificarmos.
> 
> oh, but would you accept it?

We'll accept it if everything looks good after review.

**Comment by devsuay** [comment]
> > > > Muito bom, pena que não aceitei
> > > 
> > > 
> > > Não rejeitamos, ele fechou o pull request antes de verificarmos.
> > 
> > 
> > ah, mas você aceitaria?
> 
> Aceitaremos se tudo parecer bem após a revisão.

I understood

**Pull request closed without merging**